docs/connecting-to-a-lit-network/connecting.md

```diff
@@ -0,0 +1,31 @@
+# Connecting to a Lit Network
+
+After installing the Lit SDK, you can connect an instance of [LitNodeClient](https://v6-api-doc-lit-js-sdk.vercel.app/classes/lit_node_client_src.LitNodeClient.html) to a Lit network. This is done by setting the `litNetwork` property when instantiating an instance of `LitNodeClient`:
+
+```ts
+import { LitNodeClient } from "@lit-protocol/lit-node-client";
+
+const litNodeClient = new LitNodeClient({
+    // Change this to the Lit SDK Network Identifier you want to connect to
+    litNetwork: 'datil-dev',
+});
+await litNodeClient.connect();
+```
+
+## Available Lit Networks
+
+### Mainnets
+
+| Name     | Lit SDK Network Identifier | Doc Page Link | Network is Live |
+|----------|----------------------------|---------------|-----------------|
+| Datil    | `datil`                    | n/a           | ❌               |
+| Habanero | `habanero`                 | n/a           | ✅               |
```

Comment:
i think we should add a 'notes' column to the table and emphasize that habanero, manzano, and cayenne should no longer be used and link to the migration guide

Reply:
I'm going to add a warning callout

Reply:
Addressed via [this commit](https://github.com/LIT-Protocol/docs/pull/306/commits/7c1ddd0a896b93e2bbeeb6606befc48b01cdbfeb):

<img width="1010" alt="image" src="https://github.com/LIT-Protocol/docs/assets/20375610/85d26ea7-48ff-404a-99ff-5fbfe56e0edd">